.travis.yml

```diff
@@ -21,7 +21,7 @@ before_deploy:
 - ./.travis/install_install4j
 - ENGINE_VERSION=$(grep "engine_version" game_engine.properties | sed 's/.*= *//g')
 - TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
-- ./.travis/build_installers $TAGGED_VERSION
+- TAGGED_VERSION=$TAGGED_VERSION ./gradlew release
```

Comment:
> Why is it even necessary to define a "child variable" if the actual environment variable is already set to the correct value?

Technically, there is no _environment variable_ named `TAGGED_VERSION` in the parent script; there is only a _shell variable_ of that name.

> wouldn't it just work if we exported the TAGGED_VERSION var when it's being defined in the travis script?

Yes, exporting `TAGGED_VERSION` would be functionally equivalent and is what was done prior to f687bc9.  However, as mentioned above, there was a conscious decision in f687bc9 to make the data dependencies explicit.  From that commit's message:

> Make helper scripts take parameters to show data dependency between travis.yml script and helpers

In fact, it was the implicit dependency of `build.gradle` on the `TAGGED_VERSION` environment variable that led to the build break caused by f687bc9 and fixed in 4785d02.

Since there is still some discomfort around this syntax, I will push a commit that changes `build.gradle` to not depend on an environment variable but to always use a Gradle project variable instead.